cognite/client/_api/workflows.py

```diff
@@ -47,6 +53,130 @@ def wrap_workflow_ids(
     return WorkflowIds.load(workflow_version_ids).dump(camel_case=True, as_external_id=True)
 
 
+class BetaWorkflowAPIClient(APIClient, ABC):
+    def __init__(
+        self,
+        config: ClientConfig,
+        api_version: str | None,
+        cognite_client: CogniteClient,
+    ) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._api_subversion = "beta"
+        self._warning = FeaturePreviewWarning(
+            api_maturity="beta", sdk_maturity="alpha", feature_name="Workflow Orchestration"
+        )
+
+
+class WorkflowTriggerAPI(BetaWorkflowAPIClient):
+    _RESOURCE_PATH = "/workflows/triggers"
+
+    def create(
+        self,
+        workflow_trigger: WorkflowTrigger,
+        client_credentials: ClientCredentials | None = None,
+    ) -> WorkflowTrigger:
+        """`Create a new trigger for a workflow. <https://api-docs.cognite.com/20230101/tag/Workflow-triggers/operation/createTriggers>`_
+
+        Args:
+            workflow_trigger (WorkflowTrigger): The workflow trigger specitification.
+            client_credentials (ClientCredentials | None): Specific credentials that should be used to trigger the workflow execution. When passed will take precedence over the current credentials.
+
+        Returns:
+            WorkflowTrigger: The created workflow trigger specification.
+
+        Examples:
+
+            Create a new scheduled trigger for a workflow:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.workflows import WorkflowTrigger, WorkflowScheduledTriggerRule
+                >>> client = CogniteClient()
+                >>> client.workflows.triggers.create(
+                ...     WorkflowTrigger(
+                ...         external_id="my_trigger",
+                ...         trigger_rule=WorkflowScheduledTriggerRule(cron_spec="0 0 * * *"),
+                ...         workflow_external_id="my_workflow",
+                ...         workflow_version="1",
+                ...         input_data={"a": 1, "b": 2},
+                ...     )
+                ... )
+        """
+        if workflow_trigger.authentication is None:
+            nonce = create_session_and_return_nonce(
```

Comment:
```suggestion
        if workflow_trigger.authentication is None:
            nonce = create_session_and_return_nonce(
```
Let's forego allowing setting a nonce directly similar to the .trigger() endpoint

Will need a WorkflowTriggerRead and Write model. I think that's good practice anyways, less confusion with the fields that are set by our API (lastUpdated, ...)